CONTRIBUTORS.md

```diff
@@ -4,9 +4,10 @@ Chapel Contributors
 The following people have contributed to the implementation of the
 Chapel release:
 
-Contributors to the current release
------------------------------------
+Contributors to the Chapel 1.21 and 1.22 releases
+-------------------------------------------------
 * Ben Albrecht, [HPE]
+* Sourish Ash, invidividual contributor
```

Comment:
It is Souris Ash (there's no 'h' at the end of Souris). Thanks for adding me!

Reply:
Argh, I'm glad I asked!  I don't know how I get these things so wrong.  :P

Reply:
Should be fixed now (but feel free to triple-check me).

Reply:
Yeah, it's correct now. Happens a lot with my name, no issues :-). Thanks again!